### ISSUE TITLE
2019/4/2

### ISSUE THREAD

**Issue by taxio** [StudioAquatan/TechTalkTuesday#17]
やるぞい！

前回 #16

**Comment by taxio** [comment]
GpipeとやらがOSSになったらしいので，雑に紹介する．
- https://webbigdata.jp/ai/post-3004
- https://arxiv.org/abs/1811.06965
- 公開された実装: https://github.com/tensorflow/lingvo/blob/master/lingvo/core/gpipe.py

また別の機会にlingvoの話をしたい．

**Comment by taxio** [comment]
Batch Normalizationの話もする．
- https://arxiv.org/abs/1502.03167
- https://deepage.net/deep_learning/2016/10/26/batch_normalization.html
- https://qiita.com/cfiken/items/b477c7878828ebdb0387

**Comment by pddg** [comment]
Pythonのloggingについて．以下の記事の何が良くて何がダメか

https://qiita.com/amedama/items/b856b2f30c2f38665701

**Comment by taxio** [comment]
gpipeとbatch normalizationについてはesaに書きました

**Comment by taxio** [comment]
次回 #18
